.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
       - id: codespell
         args:
             - --ignore-words-list=nin
+            - --skip=poetry.lock
```

Comment:
Some dependencies have spelling which codespell complains about, nothing we can do about that!